build/lib/packager.js

```diff
@@ -31,7 +31,7 @@ const TITANIUM_PREP_LOCATIONS = [
  */
 async function zip(cwd, filename) {
 	const command = os.platform() === 'win32' ? path.join(ROOT_DIR, 'build/win32/zip') : 'zip';
-	await exec(`${command} -9 -q -r -y "${path.join('..', path.basename(filename))}" *`, { cwd });
+	await exec(`${command} -9 -q -r "${path.join('..', path.basename(filename))}" *`, { cwd });
```

Comment:
> -y   store symbolic links as the link instead of the referenced file

it looks like this is only unavailable for windows. So we should keep that for the other platforms.

Reply:
But it wont works on windows as it

Reply:
that's why I said: `keep it for the other platforms` :wink: For windows you can remove it. Check the line above how to check for windows

Reply:
I suppose I can use **os.platform() === 'win32'** to check when to use it ?

Reply:
yes, something like `const params = os.platform() === 'win32' ? '-9 -q -r' : '-9 -q -r -y';`

Reply:
yes perfect

Reply:
I copy/paste your code 👍 and udated PR